lib/graphql/schema/validator/required_validator.rb

```diff
@@ -76,6 +77,22 @@ def validate(_object, _context, value)
             @message
           end
         end
+
+        def build_message
+          formatted_arguments = @one_of.map do |arg|
+            if arg.is_a?(Array)
+              "(" + arg.map { |a| Schema::Member::BuildType.camelize(a.to_s) }.join(" and ") + ")"
```

Comment:
Hmm, `camelize` _is_ the GraphQL-Ruby default, but plenty of people disable it. This error message should reflect `camelize: false` if it applies to these arguments. 

`@validated` will be a reference to the Field or Resolver whose arguments are being validated, so I think you could do something like: 


~~`args = @validated.all_argument_definitions`~~

Wait, even that doesn't work, because the field could have _multiple_ argument definitions with the same name. I think we'd have to build this message at runtime instead of definition time to be sure that we have the currently-active definition. Instead of returning `@message`, maybe we could build it like: 

```ruby 
arg_defns = @validated.arguments(context).values 
required_names = @one_of.map do |arg_kw| 
   # TODO handle arrays too 
   arg_defn = arg_defns.find { |defn| defn.keyword == arg_kw } 
   arg_defn.graphql_name
 end
 # then build message with `arg_names`
 ```

Reply:
Thanks for the quick review! I pushed another commit to address this feedback. (First and last commit should be combined before merging, unless you squash everything on merge, but I didn't do this yet to make it clear what changed.)